sample/src/main/java/com/esafirm/sample/CustomUIActivity.kt

```diff
@@ -87,11 +87,14 @@ class CustomUIActivity : AppCompatActivity() {
     }
 
     override fun onPrepareOptionsMenu(menu: Menu): Boolean {
+        if (config == null) super.onPrepareOptionsMenu(menu)
         val menuCamera = menu.findItem(com.esafirm.imagepicker.R.id.menu_camera)
         if (menuCamera != null) {
-            if (config != null) {
-                menuCamera.isVisible = config!!.isShowCamera
-            }
+            menuCamera.isVisible = config?.isShowCamera == true && config?.isOnlyVideo == false
+        }
+        val menuVideo = menu.findItem(com.esafirm.imagepicker.R.id.menu_video)
+        if (menuVideo != null) {
+            menuVideo.isVisible = config!!.isShowVideo
```

Comment:
IMO this should have the same pattern as menu camera check

```
menuVideo.isVisible = config?.isShowVideo == true
```

Don't forget to rename the video 😄